src/lambda/lambdaCheckParameters.ts

```diff
@@ -0,0 +1,15 @@
+import type { LambdaErrors } from './interfaces'
+
+export const lambdaCheckParameters = (object: object, indexes: string[]): object => {
+  const errors: LambdaErrors = {}
+  if (Object.keys(object).length > 0 && indexes.length > 0) {
+    indexes.forEach(index => {
+      errors[index] = { ...(!object[index] ? { error: 'undefined' } : '') }
+      // eslint-disable-next-line @typescript-eslint/no-dynamic-delete
+      if (index in object) delete errors[index]
```

Comment:
Why are u doing this verification?

Reply:
Sorry, just copied it... didn't improved it...